library/general/src/modules/PackagesProposal.rb

```diff
@@ -190,6 +193,71 @@ def RemoveResolvables(unique_id, type, resolvables, optional: false)
       true
     end
 
+    # Adds a list of resolvables to do not be installed
+    #
+    # @param [String] unique_id the unique identificator
+    # @param [Array<String>] resolvables list of resolvables to taboo
+    # @return [Boolean] whether successful
+    def AddTaboos(unique_id, resolvables)
+      return false if !CheckParams(unique_id, :package)
+
+      log.info("Taboo #{resolvables} for #{unique_id}}")
+
+      # Remove the resolvable from the list of resolvables to install
+      RemoveResolvables(unique_id, :package, resolvables)
+      RemoveResolvables(unique_id, :package, resolvables, optional: true)
```

Comment:
I would like to see comment why it needs to be called twice with different params.